src/About/About.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Super interesting, I love the way you can bring in various components.

Reply:
Material UI is looking super cool!